compendium/DeclarativeServices/src/manager/BundleLoader.cpp

```diff
@@ -78,7 +100,15 @@ GetComponentCreatorDeletors(const std::string& compName,
     try {
       auto ctx = fromBundle.GetBundleContext();
       auto opts = ctx.GetProperty(Constants::LIBRARY_LOAD_OPTIONS);
+      logger->Log(logservice::SeverityLevel::LOG_INFO,
+                  "Loading shared library for Bundle #" +
+                    ToString(fromBundle.GetBundleId()) +
+                    " (location=" + bundleLoc + ")");
       sh.Load(any_cast<int>(opts));
+      logger->Log(logservice::SeverityLevel::LOG_INFO,
+                  "Finished loading shared library for Bundle #" +
+                    ToString(fromBundle.GetBundleId()) +
+                    " (location=" + bundleLoc + ")");
     } catch (const std::system_error& ex) {
       // SharedLibrary::Load() will throw a std::system_error when a shared library
```

Comment:
It helps to also add a log when exception happens, something like "Failed loading shared library for ..."

Reply:
Thanks for the comment. This code was already merged so I will add a task for myself for a follow-up submission with this added.